pkgs/development/libraries/qt-5/5.9/qtbase/default.nix

```diff
@@ -80,7 +80,7 @@ stdenv.mkDerivation {
 
   patches =
     copyPathsToStore (lib.readPathsFromFile ./. ./series)
-    ++ stdenv.lib.optional stdenv.isDarwin ./darwin-cf.patch;
+    ++ stdenv.lib.optional stdenv.isDarwin (lib.readPathsFromFile ./. ./darwin-series);
```

Comment:
This doesn't seem to work correctly. I don't really get why it's done like that for the other patches either, what's the advantage over just a list of paths?

```
applying patch /nix/store/bz3c8wjkkippnb60hcqydklkpx6jngrm-git-export/pkgs/development/libraries/qt-5/5.9/qtbase/mkspecs-common-mac.patch
/nix/store/wh1ck71iabaw743xnf1j3aj9y8wh9jv3-stdenv-darwin/setup: line 740: /nix/store/bz3c8wjkkippnb60hcqydklkpx6jngrm-git-export/pkgs/development/libraries/qt-5/5.9/qtbase/mkspecs-common-mac.patch: No such file or directory
builder for ‘/nix/store/wbpddvjbjm993ffmcl5h2v7z3bfkhbm4-qtbase-5.9.1.drv’ failed with exit code 1
```

Reply:
> I don't really get why it's done like that for the other patches either, what's the advantage over just a list of paths?

I can point `quilt` at that list of patches and work with them on top of the unpacked sources (almost) as if it was Git.

Reply:
Oh, I just follow the current way of qt* modules. They are using a list of patches even for just one patch. Like https://github.com/NixOS/nixpkgs/tree/master/pkgs/development/libraries/qt-5/5.9/qtdeclarative

@LnL7 are you sure you pulled in all the changes? Looks like the patch file is not in your test env

Reply:
@j-hao You need to use `copyPathsToStore` as on line 82. This is probably broken for @LnL7 because he is using sandboxes, so his builder cannot access non-store paths.

Reply:
Let me fix this

Reply:
Ah yes, that must be it. The hydra builds would have the same problem.